### PR TITLE
CLI release notes v1.27.0

### PR DESCRIPTION
**Pull request by sunkickr**
CLI release notes v1.27.0 for CLI release tomorrow. We can add additional new documentation in a separate PR after the release.

### CODE REVIEW

File: astro/cli/release-notes.md [hunk 1]
```diff
@@ -18,6 +18,31 @@ This document provides a summary of all changes made to the [Astro CLI](cli/over
 
 - **Stable versions**: {{CLI_VER_LATEST}}, {{CLI_VER_2}}, and {{CLI_VER_3}}. See [Astro CLI release and lifecycle policy](cli/release-lifecycle-policy.md) for more information about support for CLI versions.
 
+## Astro CLI 1.27.0
+
+Release date: April 24, 2024
```
Comment (lzdanski) [inline astro/cli/release-notes.md:23]:
```suggestion
Release date: May 16, 2024
```

File: astro/cli/release-notes.md [hunk 2]
```diff
@@ -18,6 +18,31 @@ This document provides a summary of all changes made to the [Astro CLI](cli/over
...
+
+### New Flags for the Deployment Logs Commands: 
+
+Introducing new flags (`--webserver`, `--scheduler`, `--triggerer`, `--worker`) for the `astro deployment logs` command. Now users can parse logs for specific components, providing granular insights into Deployments and enhancing monitoring capabilities.
```
Comment (lzdanski) [inline astro/cli/release-notes.md:27]:
```suggestion
You can now parse logs for specific components, providing granular insights into Deployments and enhancing monitoring capabilities, with the following new flags for the `astro deployment logs` command:

- `--webserver`
- `--scheduler`
- `--triggerer`
- `--worker`
```

File: astro/cli/release-notes.md [hunk 3]
```diff
@@ -18,6 +18,31 @@ This document provides a summary of all changes made to the [Astro CLI](cli/over
...
+
+### Exclude DAG Files from Parse Test
```
Comment (lzdanski) [inline astro/cli/release-notes.md:29]:
```suggestion
### Exclude DAG files from parse test
```

File: astro/cli/release-notes.md [hunk 4]
```diff
@@ -18,6 +18,31 @@ This document provides a summary of all changes made to the [Astro CLI](cli/over
 
 - **Stable versions**: {{CLI_VER_LATEST}}, {{CLI_VER_2}}, and {{CLI_VER_3}}. See [Astro CLI release and lifecycle policy](cli/release-lifecycle-policy.md) for more information about support for CLI versions.
 
+## Astro CLI 1.27.0
+
+Release date: April 24, 2024
+
+### New Flags for the Deployment Logs Commands:
```
Comment (lzdanski) [inline astro/cli/release-notes.md:25]:
```suggestion
### New flags for the Deployment logs commands: 
```

File: astro/cli/release-notes.md [hunk 5]
```diff
@@ -18,6 +18,31 @@ This document provides a summary of all changes made to the [Astro CLI](cli/over
...
+
+Release date: April 24, 2024
+
+### New Flags for the Deployment Logs Commands: 
+
+Introducing new flags (`--webserver`, `--scheduler`, `--triggerer`, `--worker`) for the `astro deployment logs` command. Now users can parse logs for specific components, providing granular insights into Deployments and enhancing monitoring capabilities.
+
+### Exclude DAG Files from Parse Test
+
+Users can now exclude DAG files from testing with `astro dev parse` command. To get the new test, delete the `.astro/test_dag_integrity_default.py` file and run `astro dev init`. A new default test file will be created along with a `.astro/dag_integrity_exceptions.txt`. You can add dag files you want to be excluded to this text file. This allows you to exclude tests that you know do not pass the `astro dev parse` test.
```
Comment (lzdanski) [inline astro/cli/release-notes.md:31]:
```suggestion
You can exclude DAG files from testing with `astro dev parse` command. 

To get this new test option for an existing project, delete the `.astro/test_dag_integrity_default.py` file and run `astro dev init`. After you run this command, the Astro CLI creates a new default test file along with a `.astro/dag_integrity_exceptions.txt` text file. You can also add DAG files that you want to be excluded to the `dag_integrity_exceptions.txt` file, which allows you to exclude tests that you know do not pass the `astro dev parse` test.
```

File: astro/cli/release-notes.md [hunk 6]
```diff
@@ -18,6 +18,31 @@ This document provides a summary of all changes made to the [Astro CLI](cli/over
...
+
+### Additional improvements
+
+- The `astro deploy --image` command can now be used to deploy to deployments without pre-existing DAGs.
+- Users can now append `2>/dev/null | head` to commands to disregard upgrade messages. For example, running `astro completion bash 2>/dev/null | head`ensures that resulting bash script remains unaffected by the upgrade message.
+- New flag `astro deployment update --development-mode disable` to turn off development mode for an existing deployment. Note that you still cannot turn on development mode for an existing deployment.
+- New error message for expired API tokens.
```
Comment (lzdanski) [inline astro/cli/release-notes.md:38]:
```suggestion
- The `astro deploy --image` command can now be used to deploy to Deployments without pre-existing DAGs.
- You can now append `2>/dev/null | head` to commands to disregard upgrade messages. For example, running `astro completion bash 2>/dev/null | head` ensures that resulting bash script remains unaffected by the upgrade message.
- A new flag is available, `astro deployment update --development-mode disable`, to turn off development mode for an existing deployment. Note that you still cannot turn on development mode for an existing deployment.
```

I don't think we need to include info that there is a new error message, but if the new error message contains important information, we can add it here

File: astro/cli/release-notes.md [hunk 7]
```diff
@@ -18,6 +18,31 @@ This document provides a summary of all changes made to the [Astro CLI](cli/over
...
+
+
+### Bug fixes
```
Comment (lzdanski) [inline astro/cli/release-notes.md:41]:
```suggestion
### Bug fixes
```

File: astro/cli/release-notes.md [hunk 8]
```diff
@@ -18,6 +18,31 @@ This document provides a summary of all changes made to the [Astro CLI](cli/over
...
+
+- Fixed an issue where deployments with identical names across different workspaces couldn't be created
```
Comment (lzdanski) [inline astro/cli/release-notes.md:43]:
```suggestion
- Fixed an issue where Deployments with identical names across different Workspaces couldn't be created
```

File: astro/cli/release-notes.md [hunk 9]
```diff
@@ -18,6 +18,31 @@ This document provides a summary of all changes made to the [Astro CLI](cli/over
...
+- The upgrade-test command now returns the correct error code, ensuring accurate feedback during testing and CI/CD.
```
Comment (lzdanski) [inline astro/cli/release-notes.md:44]:
```suggestion
- The `upgrade-test` command now returns the correct error code, ensuring accurate feedback during testing and CI/CD.
```

File: astro/cli/release-notes.md [hunk 10]
```diff
@@ -18,6 +18,36 @@ This document provides a summary of all changes made to the [Astro CLI](cli/over
...
+You can now parse logs for specific components, providing granular insights into Deployments and enhancing monitoring capabilities, with the following new flags for the `astro deployment logs` command:
+
+- `--webserver`
+- `--scheduler`
+- `--triggerer`
+- `--worker`
+
+### Exclude DAG files from parse test
+
+You can exclude DAG files from testing with `astro dev parse` command.
```
Comment (lzdanski) [inline astro/cli/release-notes.md:36]:
```suggestion
You can exclude DAG files from testing with the `astro dev parse` command. 
```

File: astro/cli/release-notes.md [hunk 11]
```diff
@@ -18,6 +18,36 @@ This document provides a summary of all changes made to the [Astro CLI](cli/over
...
+- `--webserver`
+- `--scheduler`
+- `--triggerer`
+- `--worker`
+
+### Exclude DAG files from parse test
+
+You can exclude DAG files from testing with `astro dev parse` command. 
+
+To get this new test option for an existing project, delete the `.astro/test_dag_integrity_default.py` file and run `astro dev init`. After you run this command, the Astro CLI creates a new default test file along with a `.astro/dag_integrity_exceptions.txt` text file. You can also add DAG files that you want to be excluded to the `dag_integrity_exceptions.txt` file, which allows you to exclude tests that you know do not pass the `astro dev parse` test.
```
Comment (lzdanski) [inline astro/cli/release-notes.md:38]:
```suggestion
To use this new testing option for an existing project, delete the `.astro/test_dag_integrity_default.py` file and run `astro dev init`. After you run this command, the Astro CLI creates a new default test file along with a `.astro/dag_integrity_exceptions.txt` text file. You can also add DAG files that you want to be excluded to the `dag_integrity_exceptions.txt` file, which allows you to exclude tests that you know do not pass the `astro dev parse` test.
```

File: astro/cli/release-notes.md [hunk 12]
```diff
@@ -18,6 +18,36 @@ This document provides a summary of all changes made to the [Astro CLI](cli/over
...
+
+### Additional improvements
+
+- The `astro deploy --image` command can now be used to deploy to Deployments without pre-existing DAGs.
+- You can now append `2>/dev/null | head` to commands to disregard upgrade messages. For example, running `astro completion bash 2>/dev/null | head` ensures that resulting bash script remains unaffected by the upgrade message.
+- A new flag is available, `astro deployment update --development-mode disable`, to turn off development mode for an existing deployment. Note that you still cannot turn on development mode for an existing deployment.
+
+### Bug fixes
+
+- Fixed an issue where Deployments with identical names across different Workspaces couldn't be created
```
Comment (lzdanski) [inline astro/cli/release-notes.md:48]:
```suggestion
- Fixed an issue where Deployments with identical names across different Workspaces couldn't be created.
```

**Review by lzdanski — APPROVED** [review]

File: astro/cli/release-notes.md [hunk 13]
```diff
@@ -18,6 +18,36 @@ This document provides a summary of all changes made to the [Astro CLI](cli/over
...
+### Exclude DAG files from parse test
+
+You can exclude DAG files from testing with the `astro dev parse` command. 
+
+To use this new testing option for an existing project, delete the `.astro/test_dag_integrity_default.py` file and run `astro dev init`. After you run this command, the Astro CLI creates a new default test file along with a `.astro/dag_integrity_exceptions.txt` text file. You can also add DAG files that you want to be excluded to the `dag_integrity_exceptions.txt` file, which allows you to exclude tests that you know do not pass the `astro dev parse` test.
+
+### Additional improvements
+
+- The `astro deploy --image` command can now be used to deploy to Deployments without pre-existing DAGs.
+- You can now append `2>/dev/null | head` to commands to disregard upgrade messages. For example, running `astro completion bash 2>/dev/null | head` ensures that resulting bash script remains unaffected by the upgrade message.
```
Comment (jwitz) [inline astro/cli/release-notes.md:43]:
```suggestion
- You can now append `2>/dev/null | head` to commands to disregard upgrade messages. For example, running `astro completion bash 2>/dev/null | head` ensures that the resulting bash script remains unaffected by the upgrade message.
```

File: astro/cli/release-notes.md [hunk 14]
```diff
@@ -18,6 +18,36 @@ This document provides a summary of all changes made to the [Astro CLI](cli/over
...
+- A new flag is available, `astro deployment update --development-mode disable`, to turn off development mode for an existing deployment. Note that you still cannot turn on development mode for an existing deployment.
```
Comment (jwitz) [inline astro/cli/release-notes.md:44]:
```suggestion
- You can now use the `--development-mode disable` flag with `astro deployment update` to turn off [development mode](https://docs.astronomer.io/astro/deployment-resources#hibernate-a-development-deployment) for an existing Deployment. Note that you still cannot turn on development mode for an existing Deployment.
```

File: astro/cli/release-notes.md [hunk 15]
```diff
@@ -18,6 +18,36 @@ This document provides a summary of all changes made to the [Astro CLI](cli/over
...
+
+### Bug fixes
+
+- Fixed an issue where Deployments with identical names across different Workspaces couldn't be created.
```
Comment (jwitz) [inline astro/cli/release-notes.md:48]:
```suggestion
- Fixed an issue where you couldn't create two Deployments with identical names across different Workspaces. 
```

File: astro/cli/release-notes.md [hunk 16]
```diff
@@ -18,6 +18,36 @@ This document provides a summary of all changes made to the [Astro CLI](cli/over
 
 - **Stable versions**: {{CLI_VER_LATEST}}, {{CLI_VER_2}}, and {{CLI_VER_3}}. See [Astro CLI release and lifecycle policy](cli/release-lifecycle-policy.md) for more information about support for CLI versions.
 
+## Astro CLI 1.27.0
+
+Release date: May 16, 2024
+
+### New flags for the Deployment logs commands: 
+
+You can now parse logs for specific components, providing granular insights into Deployments and enhancing monitoring capabilities, with the following new flags for the `astro deployment logs` command:
```
Comment (jwitz) [inline astro/cli/release-notes.md:27]:
```suggestion
You can now filter logs for specific Deployment components using the following new flags for `astro deployment logs`:
```

Comment (jwitz) [reply astro/cli/release-notes.md:27]: Lmk what you think but I don't know what the middle clause is adding to a user's understanding of the change

**Comment by lzdanski** [comment]
Updates - https://github.com/astronomer/updates.astronomer.io/pull/218

**Pull request merged**